BTSPocket/Controllers/Profile/TableViewCells/ProfileTableViewCell.swift

```diff
@@ -28,13 +28,9 @@ class ProfileTableViewCell: UITableViewCell, MFMailComposeViewControllerDelegate
     // MARK:- Functions
     /// Set backend Profile session data in profile section
     public func loadProfile(_ backendProfileData: ProfileData?) {
```

Comment:
public statement is not needed.  backendProfileData is giving unnecessary information in the name,  rename only to 
func loadProfile(_ profile: ProfileData?)